Source/WebCore/dom/DataTransferItemList.idl

```diff
@@ -29,18 +29,18 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+// https://html.spec.whatwg.org/multipage/dnd.html#the-datatransferitemlist-interface
+
 [
     EnabledBySetting=DataTransferItemsEnabled,
     JSGenerateToNativeObject,
     JSGenerateToJSObject,
     Exposed=Window
 ] interface DataTransferItemList {
-    readonly attribute long length;
-    getter DataTransferItem item(unsigned long index);
-
+    readonly attribute unsigned long length;
+    getter DataTransferItem (unsigned long index);
```

Comment:
This also needs some kind of coverage. Perhaps a negative "historical" WPT test?